.github/actions/tests-linux-containerless/action.yaml

```diff
@@ -22,51 +22,43 @@ runs:
     with:
       distribution: 'microsoft'
       java-version: '21'
-  - run: java --version
-    shell: bash
-  - name: Set up Igor's kantra installer
+
+  - name: Get kantra bundle
+    uses: actions/download-artifact@v4
+    with:
+      name: kantra-bundle
+      path: /home/runner/.kantra
+
+  - name: Configure Test Environment
     shell: bash
     run: |
-      git clone https://github.com/ibragins/konveyor-cli-deployment.git
-      cd konveyor-cli-deployment
-      pip install -r requirements.txt
-      echo '{"misc_downstream_path": "","temp_dir": "","extract_binary": "","get_images_output": "","bundle": "","no_brew": "","ssh_user": "","ssh_key": ""}' >> config.json
-      # Workaround ruleset home (?)
-      mkdir -p /home/runner/.config
-      ./install_cli.py --upstream true
-      cd ..
       chmod +x /home/runner/.kantra/kantra
-      /home/runner/.kantra/kantra --help
+      ln -s /home/runner/.kantra /home/runner/.config/.kantra
 
-  - name: Configure Test Environment Variables
-    shell: bash
-    run: |
-      cp .env.example .env
       mkdir ${{ github.workspace }}/report
       
       > .env
-      echo "KANTRA_CLI_PATH=/home/runner/.kantra/kantra" >> .env
-      echo "REPORT_OUTPUT_PATH=${{ github.workspace }}/report" >> .env
-      echo "PROJECT_PATH=${{ github.workspace }}" >> .env
-      
+      echo "export KANTRA_CLI_PATH=/home/runner/.kantra/kantra" >> .env
+      echo "export REPORT_OUTPUT_PATH=${{ github.workspace }}/report" >> .env
+      echo "export PROJECT_PATH=${{ github.workspace }}" >> .env
```

Comment:
The env variables can be either defined in the .env file or added to the path, but in this case I think what you're doing  here is a mix of the two options that will make the `.env` file to look like this:

```
export KANTRA_CLI_PATH=/home/runner/.kantra/kantra
export REPORT_OUTPUT_PATH=workspace/report
export PROJECT_PATH=workspace
```

instead of 

```
KANTRA_CLI_PATH=/home/runner/.kantra/kantra
REPORT_OUTPUT_PATH=workspace/report
PROJECT_PATH=workspace
```

Reply:
Good point, thanks. Updated unifying to exported variables instead of file as already used for windows workflow.